client/my-sites/sites/sites.jsx

```diff
@@ -93,4 +108,13 @@ export default React.createClass( {
 			</Main>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state ) => ( {
```

Comment:
You're right @nb this is actually a problem. @aduth Is there a reason why we don't sync all sites in `state.sites` ?